### ISSUE TITLE
Issue connecting Trezor One with firmware 1.10.X

### ISSUE THREAD

**Issue by barbozadev** [walleth/walleth#546]
Hi guys! 
Error connecting Trezor One with firmware 1.10.3, seems that the app is just supporting 1.8.X or 1.9.X so far.

![photo_2021-09-27_02-58-16](https://user-images.githubusercontent.com/53384914/134877367-58549d7e-4ab5-4167-932c-d6f0fa50922e.jpg)

**Comment by GuizzyQC** [comment]
This was resolved in 0.51.0, although that version hasn't been deployed to the Google Play Store yet, which is why you are still experiencing the issue. You can get a more up-to-date version from F-Droid.

**Comment by barbozadev** [comment]
Yes, It works with F-Droid! But, using Trezor One which is my case, in the last confirm interaction sending a transaction nothing happened, I mean I pressed the confirm button on Trezor and stayed in the same screen turned on with the confirm and cancel button, in the phone just shows the screen that request to sign in the Trezor. So in the last step sending a transaction there's a critical bug when pressing the confirm button.

**Comment by AarbozT** [comment]
> This was resolved in 0.51.0, although that version hasn't been deployed to the Google Play Store yet, which is why you are still experiencing the issue. You can get a more up-to-date version from F-Droid.

Yes, It works with F-Droid! But, using Trezor One which is my case, in the last confirm interaction sending a transaction nothing happened, I mean I pressed the confirm button on Trezor and stayed in the same screen turned on with the confirm and cancel button, in the phone just shows the screen that request to sign in the Trezor. So in the last step sending a transaction there's a critical bug when pressing the confirm button.
